sdk/js/src/Client.ts

```diff
@@ -49,13 +48,9 @@ export class DVCClient implements Client {
 
     constructor(environmentKey: string, user: DVCUser, options: DVCOptions = {}) {
         this.logger = options.logger || dvcDefaultLogger({ level: options.logLevel })
-        if (options?.reactNative) {
-            this.store = new ReactNativeStore(this.logger)
-        } else {
-            this.store = new Store(
-                typeof window !== 'undefined' ? window.localStorage : stubbedLocalStorage, this.logger
-            )
-        }
+        this.store = new Store(
+            typeof window !== 'undefined' ? window.localStorage : stubbedLocalStorage, this.logger
```

Comment:
we shouldn't need the stubbedLocalStorage anymore

Reply:
its used for server side rendering, when there's no `window.localStorage` available to make sure the app doesn't crash